src/index.js

```diff
@@ -242,90 +195,91 @@ async function findVolunteers(request) {
 async function checkForNewSubmissions() {
   base('Requests').select({ view: 'Grid view' }).eachPage(async (records, nextPage) => {
     // Look for records that have not been posted to slack yet
-    await asyncForEach(records, async (record) => {
-      if (record.get('Posted to Slack?') === 'yes') { return; }
-      log(`\nProcessing: ${record.get('Name')}`);
-
-      // Prepare the general info
-      const profileURL = `https://airtable.com/tblaL1g6IzH6uPclD/viwEjCF8PkEfQiLFC/${record.id}`;
-      const header = [
-        `<${profileURL}|${record.get('Name')}>`,
-        record.get('Phone number'),
-        record.get('Address'),
-      ];
-      const errandObject = {
-        type: 'section',
-        text: {
-          type: 'mrkdwn',
-          text: header.join('\n'),
-        },
-      };
-
-      // Process the requested tasks
-      const taskObject = {
-        type: 'section',
-        text: {
-          type: 'mrkdwn',
-          text: `*Needs assistance with:*${formatTasks(record)}`,
-        },
-      };
+    for (const record of records) {
```

Comment:
so much nicer 😭

Reply:
Haha no need for multiple layers of promises